ftw/linkchecker/browser/dashboard.py

```diff
@@ -1,5 +1,34 @@
+from plone import api
+from zope.annotation.interfaces import IAnnotations
 from zope.publisher.browser import BrowserView
 
 
 class Dashboard(BrowserView):
+
+    def __init__(self, context, request):
+        super(Dashboard, self).__init__(context, request)
+        self.dashboard_model = DashboardModel(self.context, self.request)
+        self.graph_generator = GraphGenerator()
```

Comment:
This pull request is all about storage and its configuration. What is the purpose of the graph generator in this topic?

Reply:
It is preparing the setup like in the concept part of the IPA. It follows the pattern below, trying to prepare all parts of the architecture of the browser package.

![image](https://user-images.githubusercontent.com/29920936/80471775-2cc34e80-8944-11ea-90d3-905d16f462b4.png)

Reply:
Ok